### DISCUSSION TITLE
How to get list of resources backup by Velero

### DISCUSSION THREAD

**Discussion by TechGirl-3** [vmware-tanzu/velero#6271] [Community support Q&A]
Hi Team,

Could you point me to where I can check the list of resources that are backup by Velero? Also is there a document that lists the resources being backup by Velero and the order of backup?

Thank You.

**Comment by draghuram** [comment]
Hi,

You can run the command `velero backup describe --details <BACKUP-NAME>"` to see the full list of resources in the backup. Velero backs up all the resources on the cluster depending on the selection and whether you included cluster scoped resources or not. 

I will try to find documentation that refers to resource order of the backup but note that you can control order of the backup by by using "--ordered-resources" option to "velero backup create".

Hope this helps, Raghu.

**Reply by varac** [reply]
`velero backup describe --details <BACKUP-NAME>"` only shows the Kubernetes cluster resources, but not the files beeing backed up by a filesystem backup i.e.
I'm also looking for an advice how to list all files in a backup set.
